libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Version history for `cardano-ledger-binary`
 
-## 1.3.2.1
+## 1.3.3.0
```

Comment:
The version in the cabal file itself should also be updated, no?

Reply:
Thanks for reminding